src/Whoops/Resources/js/whoops.base.js

```diff
@@ -119,4 +119,10 @@ Zepto(function($) {
     e.preventDefault();
     $.get(this.href);
   });
+
+  // Symfony VarDumper: Close the by default expanded objects
+  $('.sf-dump-expanded')
+    .removeClass("sf-dump-expanded")
+    .addClass("sf-dump-compact");
+  $(".sf-dump-toggle span").html('▶');
```

Comment:
please use a html sequence instead of a UTF8 char (to make it work on non utf8 websites)